Resolve rubocops

kbrock opened the pull request:
Now may be a good time to push these - as we will be ramping up our testing

None of these overlap with the active record 7.0 changes

I skipped a number of warnings around parenthesis. I felt they assisted the code. Also avoided changing the variable `to`, as with delegation, that is the correct phrase.

```
< 32 files inspected, 87 offenses detected, 58 offenses autocorrectable
---
33 files inspected, 51 offenses detected, 27 offenses autocorrectable
```

Fryguy commented on lib/active_record/virtual_attributes/virtual_delegates.rb:
```diff
@@ -125,7 +126,7 @@ def #{method_name}(#{definition})
         end
 
-        def virtual_delegate_name_prefix(prefix, to)
+        def virtual_delegate_name_prefix(prefix, to)  # rubocop:disable Naming/MethodParameterName
```
I'm surprised this didn't trigger another rubocop with the 2 spaces before the comment.

kbrock replied:
lol. it did trigger that

Fryguy commented on lib/active_record/virtual_attributes/virtual_total.rb:
```diff
@@ -49,35 +49,6 @@ def virtual_average(name, relation, column, options = {})
-        return virtual_total(name, relation, options) if method_name == :size
-        return virtual_sum(name, relation, column, options) if method_name == :sum
-      end
-
```
This seems like an actual code change.

kbrock replied:
I'll pull into another PR

I think it was never used - and if it was, transitioned in our 6.0 upgrade.
grepping the code shows up empty.

Fryguy commented on spec/virtual_includes_spec.rb:
```diff
@@ -224,7 +224,10 @@
   it "preloads virtual_attribute in :include when :conditions are also present in calculations" do
     expect(Book.includes([:author_name, :author]).references(:author).where("authors.name = '#{author_name}'")).to preload_values(:author_name, author_name)
     expect(Book.includes([:author_name, :author]).references(:author).where(:authors => {:name => author_name})).to preload_values(:author_name, author_name)
+    # specifically, we are specifically testing this use case
```
redundant wording...maybe
```suggestion
    # Disable Rails/WhereNot because we are testing this specific use case
```

Fryguy commented:
Merged #152 152, so please rebase.

kbrock commented:
update
- rebased
- extracted change
- applied changes per comments
- fixed spelling for rubocop:disable Style/SingleLineMethods
- dropped Naming/AccessorMethodName (no longer needed)
